<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modulesTestDMXProjectClass</class>
 <widget class="QMainWindow" name="modulesTestDMXProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMXProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>8</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>1020</width>
          <height>283</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="9" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Liste des scenes existantes :</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="pushButtonValider">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Valider</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditNomScene">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelNomScene">
           <property name="text">
            <string>Cr√©er une nouvelle scene : </string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1181</width>
          <height>661</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Selectionnez vos equipements :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutEquipements">
           <item>
            <widget class="QCheckBox" name="widgetEquipements">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Selectionnez quelle scene vous souhaitez configurer :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutScenes">
           <item>
            <widget class="QCheckBox" name="widgetScenes">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>1220</x>
          <y>660</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>901</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="nomEquipLabel_2">
               <property name="text">
                <string>Nom de l'equipement :</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="adresseEquipLabel">
               <property name="text">
                <string>Adresse de l'equipement :</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="nbCanauxLabel">
               <property name="text">
                <string>Nombre de canaux :</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="buttonEquip">
               <property name="text">
                <string>Ajouter equipement</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLineEdit" name="nomEquipEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLineEdit" name="adresseEquipEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLineEdit" name="nbCannauxEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>491</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ajouter un Equipement</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>861</width>
          <height>541</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17"/>
       </widget>
       <widget class="QPushButton" name="validateButtonEquip">
        <property name="geometry">
         <rect>
          <x>1100</x>
          <y>600</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>80</y>
          <width>791</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18"/>
       </widget>
       <widget class="QPushButton" name="ValidateButtonCanal">
        <property name="geometry">
         <rect>
          <x>1080</x>
          <y>540</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>1051</width>
          <height>631</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QComboBox" name="sceneComboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>290</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>260</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Choisir une scene a tester : </string>
        </property>
       </widget>
       <widget class="QPushButton" name="testSceneButton">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>290</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Tester la scene</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>10</y>
          <width>241</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DMX Project</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>670</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>creer par: LOPES Damien ; BOTEL Faustin ; LEDUC Jean ; LEFEVRE Tom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>270</y>
          <width>271</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QLabel" name="champLabel">
        <property name="geometry">
         <rect>
          <x>146</x>
          <y>500</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom du champ :</string>
        </property>
       </widget>
       <widget class="QLabel" name="nomSceneLabel">
        <property name="geometry">
         <rect>
          <x>176</x>
          <y>400</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom de la scene :</string>
        </property>
       </widget>
       <widget class="QLabel" name="nomSceneAfficheLabel">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>400</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="valeurAfficheLabel">
        <property name="geometry">
         <rect>
          <x>256</x>
          <y>520</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="validateCanalButton">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>640</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QLabel" name="sceneLabel">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Choisir une scene</string>
        </property>
       </widget>
       <widget class="QComboBox" name="SceneComboBox">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>40</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="equipLabel">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>40</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Choisir un equipement</string>
        </property>
       </widget>
       <widget class="QComboBox" name="EquipComboBox">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>40</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ValidSceneEquipButton">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>450</y>
          <width>701</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutSlideBar">
         <item>
          <widget class="QSlider" name="verticalSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChoisir">
    <property name="title">
     <string>Choisir</string>
    </property>
    <addaction name="actionSupprimer_un_equipement"/>
    <addaction name="actionCreer_une_sc_ne"/>
    <addaction name="actionConfigurer_une_sc_ne_2"/>
    <addaction name="actionTester_une_scene"/>
    <addaction name="actionArduino"/>
   </widget>
   <addaction name="menuChoisir"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreer_une_sc_ne">
   <property name="text">
    <string>Creer une scene</string>
   </property>
  </action>
  <action name="actionConfigurer_une_sc_ne_2">
   <property name="text">
    <string>Configurer une scene</string>
   </property>
  </action>
  <action name="actionAjouter_un_equipement">
   <property name="text">
    <string>Ajouter un equipement</string>
   </property>
  </action>
  <action name="actionSupprimer_un_equipement">
   <property name="text">
    <string>Gerer un equipement</string>
   </property>
  </action>
  <action name="actionTester_une_scene">
   <property name="text">
    <string>Tester une scene</string>
   </property>
  </action>
  <action name="actionArduino">
   <property name="text">
    <string>Arduino</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="modulesTestDMXProject.qrc"/>
 </resources>
 <connections/>
</ui>
